featuretools/tests/entityset_tests/test_es.py

```diff
@@ -32,8 +32,10 @@ def test_normalize_time_index_as_additional_variable(es):
 def test_cannot_re_add_relationships_that_already_exists(es):
     warn_text = "Not adding duplicate relationship: " + str(es.relationships[0])
     before_len = len(es.relationships)
+    rel = es.relationships[0]
     with pytest.warns(UserWarning, match=warn_text):
-        es.add_relationship(es.relationships[0])
+        es.add_relationship(rel._parent_dataframe_id, rel._parent_column_id,
```

Comment:
```
es.add_relationship(relationship=es.relationships[0])
```

would be shorter

since we're testing the re-add validation specifically i don't think which input we use matters

or we could test both

Reply:
I think the way things are structured now the input doesn't matter, but if we ever changed the way we check for an existing relationship it might. With that in mind, I think it might be good to test both approaches here.